### ISSUE TITLE
XHTML: Image path become invalid

### ISSUE THREAD

**Issue by ToshihikoMakita** [dita-ot/dita-ot#2568]
When I build the attached DITA instance [20170114-diff.zip](https://github.com/dita-ot/dita-ot/files/704702/20170114-diff.zip) , it finished normally. But the generated image path become invalid and image is not displayed.

[Command-line]
```
bin\dita -i D:\SVN\pdf5\diffs\20170114-diff\ja-JP\bmAcme.ditamap -f xhtml -o D:\SVN\pdf5\diffs\20170114-diff\ja-JP\out --generate.copy.outer=3 --clean.temp=no --dita.temp.dir=D:\SVN\pdf5\diffs\20170114-diff\ja-JP\temp\xhtml --logfile=D:\SVN\pdf5\diffs\20170114-diff\ja-JP\log.txt -v
```

[The relevant location: ja-JP/out/ja-jP/topic/cHelpWindow.html line 28]

```
<img class="image" id="concept_HelpWindow__image_dpc_gyn_vx" src="../../../common/image/001_01_3.0.gif" />
```

should be 

```
<img class="image" id="concept_HelpWindow__image_dpc_gyn_vx" src="../../common/image/001_01_3.0.gif" />
```

Please fix this bug.

Regards,

-- 
/*--------------------------------------------------
 Toshihiko Makita
 Development Group. Antenna House, Inc. Ina Branch
 Web site:
 http://www.antenna.co.jp/
 http://www.antennahouse.com/
 --------------------------------------------------*/

**Comment by robander** [comment]
I've reproduced this with 2.4.3 -- I wonder if it's related to a couple of other issues (most recently #2581) where relative paths to local non-DITA resources in key definitions are not properly adjusted. Although in other cases that typically results in paths _not_ getting adjusted -- while in this case I think it's adjusted too much.

**Comment by mironovalexey** [comment]
Probably resolved: https://github.com/dita-ot/dita-ot/commit/3612e55aab141e664a78c2720e65b8a5759eae9b

**Comment by jelovirt** [comment]
@mironovalexey are you able to verify that this is fixed in `develop`?

**Comment by mironovalexey** [comment]
Yes. It's ok now.  The result is in the attachment (fresh dita-ot build).
[test.tar.gz](https://github.com/dita-ot/dita-ot/files/950526/test.tar.gz)

**Comment by jelovirt** [comment]
Fixed by #2667
